reactioncommerce/reaction
# fix: issue that prevented uploading images on Firefox

Signed-off-by: Will Lopez <will.lopez77@gmail.com>

Resolves #5840 
Impact: Major  
Type: Bugfix

## Issue
Users were not able able to upload product images on Firefox due to an issue with the reaction file collections package. The issue was has been fixed there, this PR updates Reaction to bring in those updates.

## Solution
Update `reaction-file-collections` package to use the latest version.

## Testing
1. Use Firefox to upload a product image
2. Verify image uploads correctly
